client/src/components/ProductDetails/ProductDetails.jsx

```diff
@@ -0,0 +1,67 @@
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import ProductDetailsHeading from './ProductDetails_Heading/ProductDetails_Heading';
+import ProductDetailsPolicy from './ProductDetails_Policy/ProductDetails_Policy';
+import './styles.scss';
+import { convertMoney } from '../../helpers/convertMoney';
+
+const ProductDetails = () => {
+  const { id } = useParams();
+  const { products } = useSelector(state => state.products)
+
+  const product = products.find(product => product.id === +id);
+
+  const renderItinerary = (itinerary) => {
+    const processedIti = itinerary.split('/n');
+
+    return (
+      <div>
+        <img src={product.mainImg} alt='productDetails-img'></img>
+        <ProductDetailsHeading text='chương trình tour' />
+        {processedIti.map(sentence => {
+          if(sentence.includes("NGÀY")){
+            return <h6>{sentence}</h6>
+          } else if(sentence.includes("https")){
+            return <img src={sentence} alt='product-img'></img>
+          } else {
+            return <p>{sentence}</p>
+          }
+        })}
+      </div>
```

Comment:
Đống gì đây anh, content của tour ấy hả? Cái này anh làm tạm thời thì được
Chứ còn về sau khi làm admin thì đống content này nó sẽ dưới dạng
```
{
   id: '',
   content: '<div>content 1</div><div>content 2</div>'
}
```
Rồi in ra bằng `dangerouslySetInnerHtml`
đống content này sẽ được generate ra bởi rich-text editor. Anh có thể tìm hiểu sau nhé, vd: ckeditor, quill, lexical, draftjs,...

Reply:
ừa ok ok, tại a kb, để a tìm hiểu thêm. Hiện giờ thì a vẫn dùng kiểu này được đúng k.